packages/modules/itwin-viewer-react/src/services/Initializer.ts

```diff
@@ -105,11 +107,53 @@ class Initializer {
     }
   }
 
+  public static async getSynchronizerUrl(
+    contextId: string,
+    iModelId: string
+  ): Promise<string> {
+    if (!this._synchronizerRootUrl) {
+      const urlDiscoveryClient = new UrlDiscoveryClient();
+      this._synchronizerRootUrl = await urlDiscoveryClient.discoverUrl(
+        new ClientRequestContext(),
+        "itwinbridgeportal",
+        Config.App.get("imjs_buddi_resolve_url_using_region")
```

Comment:
i think you can actually pass undefined here, which should default to using the value for imjs_buddi_resolve_url_using_region under the hood

Reply:
thanks. I'll update